### PR TITLE
Porting Bridge (Sethu) Module

### PR DESCRIPTION
**Pull request by murthyvitwit**
#443

### CODE REVIEW

File: bridge/setu/broadcaster/broadcaster.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	if err := maticClient.SendTransaction(context.Background(), signedTx); err != nil {
+		tb.logger.Error("Error while broadcasting the transaction to maticchain", "error", err)
+		return err
+	}
+
+	return nil
+}
+
+// BroadcastToRootchain broadcast to rootchain
+func (tb *TxBroadcaster) BroadcastToRootchain() {}
```
Comment (ssandeep) [inline bridge/setu/broadcaster/broadcaster.go:142]: We'll implement this later?

File: scripts/protoc-swagger-gen.sh [hunk 2]
```diff
@@ -22,7 +22,7 @@ done
 # combine swagger files
 # uses nodejs package `swagger-combine`.
 # all the individual swagger files need to be configured in `config.json` for merging
-swagger-combine ./client/docs/config.json \
+/home/gsk967/.nvm/versions/node/v10.23.1/bin/swagger-combine ./client/docs/config.json \
```
Comment (ssandeep) [inline scripts/protoc-swagger-gen.sh:25]: we'll have to change back this one

File: types/rest/rest.go [hunk 3]
```diff
@@ -37,12 +37,25 @@ var (
 // ResponseWithHeight defines a response object type that wraps an original
 // response with a height.
 type ResponseWithHeight struct {
-	Height int64           `json:"height"`
+	Height string          `json:"height"`
```
Comment (ssandeep) [inline types/rest/rest.go:40]: why are we changing the datatype of height?

**Review by ssandeep — CHANGES_REQUESTED** [review]

File: bridge/setu/broadcaster/broadcaster.go (hunk 1 above)
Comment (saikumar0x1967) [reply bridge/setu/broadcaster/broadcaster.go:142]: no, it is like that in v0.3.0 version 
https://github.com/maticnetwork/heimdall/blob/e58da51d56857bd3d2f61c8e7b22f4ff024bac75/bridge/setu/broadcaster/broadcaster.go#L141

File: scripts/protoc-swagger-gen.sh (hunk 2 above)
Comment (saikumar0x1967) [reply scripts/protoc-swagger-gen.sh:25]: ok

**Comment by saikumar0x1967** [comment]
@ssandeep  can you review it again, i done review changes

**Review by ssandeep — APPROVED** [review]

**Pull request merged**